andela/travel_tool_front
# #161510204 Fix comment action buttons

#### What does this PR do?
- References email in the store to display edit and delete button in a comment header

#### Description of Task to be completed?
- retrieve email accurately from the store

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- Checkout to the branch with git fetch -a then git checkout bg-fix-comment-action-buttons-161510204
- Pull develop on the backend
- Run the server with make start
- Navigate to My Requests page and create a new request
- Add a comment to the just created request.
- Edit and Delete button should be available in the comment header

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161510204](https://www.pivotaltracker.com/story/show/161510204)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-26 at 4 57 51 pm" src="https://user-images.githubusercontent.com/23608060/47578204-b3a6fd80-d940-11e8-9e98-108fa0fb1a44.png">

#### Questions:
N/A
